com.blitterstudio.amiberry.yml

```diff
@@ -0,0 +1,89 @@
+app-id: com.blitterstudio.amiberry
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: amiberry.sh
+rename-icon: amiberry
+copy-icon: true
+finish-args:
+  - --device=all
+  # X11 + XShm access
+  - --share=ipc
+  # Wayland access
+  - --socket=wayland
+  - --socket=fallback-x11
+  # Needs to talk to the network:
+  - --share=network
+  # Allow bluetooth (e.g. controllers)
+  - --allow=bluetooth
+  # Play sounds using PulseAudio
+  - --socket=pulseaudio
+  # Filesystem access
+  - --filesystem=host
+  
+modules:
+  - name: libmpeg2
+    config-opts:
+      - --disable-static
+    rm-configure: true
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - "*.la"
+      - "*.a"
+    sources:
+      - type: archive
+        url: http://libmpeg2.sourceforge.net/files/libmpeg2-0.5.1.tar.gz
+        sha256: dee22e893cb5fc2b2b6ebd60b88478ab8556cb3b93f9a0d7ce8f3b61851871d4
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -vfi
+
+  - name: libserialport
+    buildsystem: autotools
+    cleanup:
+      - /include
+      - "*.la"
+      - "*.a"
+    sources:
+      - type: git
+        url: https://github.com/sigrokproject/libserialport
+        tag: libserialport-0.1.1
+        commit: 348a6d353af8ac142f68fbf9fe0f4d070448d945
+
+  - name: amiberry
+    only-arches:
+      - x86_64
+    post-install:
+      - install -Dp -m 755 amiberry /app/bin/amiberry
+      - install -D -m644 -t /app/share/applications/ flatpak/${FLATPAK_ID}.desktop
+      - install -D -m644 -t /app/share/metainfo/ flatpak/${FLATPAK_ID}.metainfo.xml
+      - install -D -m755 -t /app/bin/ ./amiberry.sh
+      - install -Dm644 flatpak/256x256.png /app/share/icons/hicolor/256x256/apps/amiberry.png
+      - mkdir -p /app/data
+      - cp -R abr /app/data
+      - cp -R conf /app/data
+      - cp -R controllers /app/data
+      - cp -R data /app/data
+      - cp -R inputrecordings /app/data
+      - cp -R kickstarts /app/data
+      - cp -R nvram /app/data
+      - cp -R savestates /app/data
+      - cp -R screenshots /app/data
+      - cp -R whdboot /app/data
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    
+    sources:
+      - type: git
+        url: https://github.com/BlitterStudio/amiberry
+        tag: v5.6.2
```

Comment:
content_rating isn't part of this tag https://github.com/BlitterStudio/amiberry/commit/cbd437d7192eee6a4bb495ccba6d710edbbd0c83

It should be `oars-1.1` too

Reply:
Yeah, I noticed - I've been preparing a new release/tag in the meantime, so I'll ping back here when that's ready :)

Reply:
I've updated it to the new tag, which should work now